macroid-docs/guide/Tweaks.md

```diff
@@ -115,6 +115,50 @@ A quick rundown:
         ...
       }
     ```
+  * `MultiOn` — works for the event listeners with several methods to override. 
+  
+     It must be used as follows:

```

Comment:
Looks like this line and some lines below are mis-indented by one space :) Not that it changes the HTML output (I checked), but it is a bit disturbing :)